Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -35,6 +35,14 @@ DECLARE_SOA_TABLE(BCs, "AOD", "BC", o2::soa::Index<>,
                   bc::TriggerMask);
 using BC = BCs::iterator;
 
+namespace ts
+{
+DECLARE_SOA_INDEX_COLUMN(BC, bc);
```

Comment:
The index can be removed. You use it with join later, so it is mapped 1-1 to the BC table (each BC entry has one corresponding Timestamp entry)

Reply:
Ok, done